zerver/views/__init__.py

```diff
@@ -572,6 +572,31 @@ def dev_direct_login(request, **kwargs):
     return HttpResponseRedirect("%s%s" % (settings.EXTERNAL_URI_SCHEME,
                                           request.get_host()))
 
+@csrf_exempt
+@require_post
+@has_request_variables
+def api_dev_android_direct_login(request, username=REQ):
+    # This function allows logging in without a password on Android app and should only be called in development
+    # environments. It may be called if the DevAuthBackend is included in settings.AUTHENTICATION_BACKENDS
+    if not dev_auth_enabled() or settings.PRODUCTION:
+        return json_error(_("Dev environment not enabled.", data={"reason": "dev_disabled"}, status=403))
+    user_profile = authenticate(username=username)
+    login(request, user_profile)
+    if user_profile.realm.deactivated:
+        raise json_error(_("Realm for account has been deactivated"))
+    if not user_profile.is_active:
+        return json_error(_("Your account has been disabled.", data={"reason": "disabled"}, status=403))
+    return json_success({"api_key": user_profile.api_key, "email": user_profile.email})
+
+@csrf_exempt
+def api_android_get_dev_email(request):
+    if not dev_auth_enabled() or settings.PRODUCTION:
+        return json_error(_("Dev environment not enabled.", data={"reason": "dev_disabled"}, status=403))
+    MAX_DEV_BACKEND_USERS = 100
+    users_query = UserProfile.objects.select_related().filter(is_bot=False, is_active=True)
+    users = users_query.order_by('email')[0:MAX_DEV_BACKEND_USERS]
+    return json_success(dict(direct_admins=[u.email for u in users if u.is_realm_admin], direct_users=[u.email for u in users if not u.is_realm_admin]))
```

Comment:
you should line wrap `direct_users` to be on the next line, and aligned (using spaces, not tabs) with `direct_admins`